tests/integration/tier1/remove-all-submgr-pkgs/test_remove_all_submgr_pkgs.py

```diff
@@ -22,14 +18,11 @@ def test_remove_all_submgr_pkgs(shell, convert2rhel):
             env.str("RHSM_POOL"),
         )
     ) as c2r:
-        if "Oracle Linux Server release 7" in system_release:  # We're not installing sub-mgr on OL 7
```

Comment:
Since we remove `rhn-client-tool` in general ansible role (https://github.com/oamg/convert2rhel/pull/310/files#diff-effa0515e248da3fe738adfe9c21f2977701c9e1c8701bcc86c36acb32415a43R83), the sub-manager package is being installed on OL7 as well. So we do not need to have this special condition anymore.

Reply:
The issue is that we want to test a scenario where we do not install pakcage on OL7 intentionally. I will address this in other PR.